packages/edit-site/src/components/sidebar-navigation-screen-navigation-menus/index.js

```diff
@@ -17,7 +17,7 @@ export default function SidebarNavigationScreenNavigationMenus() {
 			title={ __( 'Navigation' ) }
 			content={
 				<div className="edit-site-sidebar-navigation-screen-navigation-menus">
-					<NavigationInspector />
+					<NavigationInspector enableNavigation={ true } />
```

Comment:
This is a weird way to implement this feature. Wouldn't it be better to have an `onSelect` here instead. It would keep the `OffCanvasEditor` independent from the "url"/"site editor".

Reply:
> OffCanvasEditor independent from the "url"/"site editor"

OffCanvasEditor was already independent. We are not changing anything of OffCanvasEditor in this PR. NavigationInspector is a site editor component used just on the site editor, but your suggestion is great, and I applied it to  NavigationInspector, which now receives an onSelect.